tests/helper/odo_utils.go

```diff
@@ -59,6 +59,24 @@ func DetermineRouteURL(context string) string {
 	return routeURL(context)
 }
 
+// DetermineRouteURLs takes context path as argument and returns the URLs
+// where the current component exposes it's service, these URL can
+// then be used in order to interact with the deployed service running in Openshift
+func DetermineRouteURLs(context string) []string {
```

Comment:
There already is an almost identical function `routeURL` .
This creates duplicated code.
Would it make sense to delete the old `routeURL` and use new `DetermineRouteURLs` everywhere?

Reply:
I will do this in a follow up

Reply:
why? You are adding a duplicate function just so you can later delete it?

Reply:
This is not a duplicate function - this one returns list of URLs, `routeURL` returns just the first one, so if you want me to use `DetermineRouteURLs` then that would require work, its not a simple search and replace

Reply:
And as the function signature is different I cannot breakout the code into a helper function as well.

Reply:
Both functions servered a different purpose but I still removed duplicate code.